src/add_remove.test.js

```diff
@@ -12,6 +12,18 @@ describe('Add, Remove Element Test', () => {
     expect(JSON.parse(localStorage.getItem('List'))).toHaveLength(1);
   });
 
+  test('Add one new item to the list', () => {
+    document.body.innerHTML = '<div class="list-elements">'
+
+    + '</div>';
+    addElement('Cooking');
+    const list = document.querySelectorAll('.list-elements');
+    expect(list).toHaveLength(1);
+    // expect(list).toHave(`
```

Comment:
- [ ] Kindly consider removing all unesessery commented lines of code from your code, so it's clean and more readable

Reply:
@Ismailco Thanks we on it

Reply:
Well resolved 👍🏽